### PR TITLE
Fixes #20612 - Fix error rendering when building pxe default

### PR DESCRIPTION
**Pull request by amirfefer**

### CODE REVIEW

**Comment by ohadlevy** [comment]
can you please add a test? thanks

**Comment by amirfefer** [comment]
@ohadlevy I've added a test case.

File: test/controllers/provisioning_templates_controller_test.rb [hunk 1]
```diff
@@ -133,6 +133,7 @@ class ProvisioningTemplatesControllerTest < ActionController::TestCase
     test "build menu" do
       ProxyAPI::TFTP.any_instance.expects(:create_default).with(regexp_matches(/^PXE.*/), has_entry(:menu, regexp_matches(/ks=http:\/\/foreman.unattended.url\/unattended\/template/))).returns(true).times(3)
       get :build_pxe_default, {}, set_session_user
+      assert flash[:notice].present?
```
Comment (ohadlevy) [inline test/controllers/provisioning_templates_controller_test.rb:136]: maybe add that flash[:warning (or failure)?] is empty?

**Comment by amirfefer** [comment]
[test foreman]

**Comment by amirfefer** [comment]
@ohadlevy, thanks.
I've added `flash[:error]` test case

**Comment by tbrisker** [comment]
Thanks @amirfefer !

**Pull request merged**